instructor_notes/openscpca/introduce-analysis-modules.md

```diff
@@ -0,0 +1,89 @@
+# Instructor Notes: Introduction to analysis modules Activity
+
+
+## Learning goals
+
+- Understand the structure of `OpenScPCA-analysis` analysis modules
+- Learn how to explore and run modules, including environment setup
+- Gain experience with the data download script
+
+## Activity
+
+Instructors will explain and demonstrate this activity while participants follow along.
+We expect that participants will take all steps along with the instructor.
```

Comment:
Just as a question: which environment are we planning to use here? VSCodium?

Reply:
Yeah, that was my thought (see #160), since we'll be in python land